lib/agent.js

```diff
@@ -64,7 +64,9 @@ class Agent {
                 agent.connected = true
                 console.log(`connected to ${agent.creds.host}:${agent.creds.port} as ${options.clientId}`)
                 agent.error = null
-                agent.client.subscribe('#')
+                const topic = agent.creds.topicPrefix || '#'
+                console.log(`subscribing to ${topic}`)
```

Comment:
```suggestion
```

Is this deliberate - for logging purposes Ben?

Reply:
Intended, this how the agent logs